packages/compartment-mapper/test/fixtures-cjs-compat/node_modules/app/index.js

```diff
@@ -0,0 +1,48 @@
+const callself = require('callself')
+const interops = require('interops')
+const moduleinterops = require('./moduleinterops')
+
+// generated by running this file with console.error(stableOrderJson(interops))
+const whatWouldNodejsRequireDo = '{"default":{"nameCollision":1},"nameCollision":3,"notDefault":2}'
+// Would we rather reproduce the import behavior? see _generateExpected.mjs
+// const whatWouldNodejsImportDo = '{"default":{"default":{"nameCollision":1},"nameCollision":3,"notDefault":2},"nameCollision":3,"notDefault":2}'
+
+
+
+function assertInteropNameCollisions({ moduleReference, expect, name }) {
+  const whatWeGot = stableOrderJson(moduleReference)
+  if (whatWeGot !== expect) {
+    throw Error(`Exporting a field called 'default' from a cjs module '${name}' should not override all other fields.
+    What we got     : ${whatWeGot}
+    What we expected: ${expect}`)
+  }
+}
+
+function stableOrderJson(obj) {
+  return JSON.stringify(obj, Object.keys(obj).sort());
+}
+
+
```

Comment:
Our prettifier would probably collapse these lines. It might not be reaching here because of the `node_modules` path component. Maybe we can work-around that somehow without interfering with the test fixture shape, but I don’t know. Fixing the linter is out of scope for this change.

Reply:
It was less work than finding what to fix